includes/admin/class-wc-rest-stripe-settings-controller.php

```
@@ -280,9 +280,12 @@ public function update_settings( WP_REST_Request $request ) {
 		$this->update_is_test_mode_enabled( $request );
 
 		/* Settings > Payments accepted on checkout + Express checkouts */
-		$payment_method_ids_to_enable = $this->get_payment_method_ids_to_enable( $request );
-		$is_upe_enabled               = $request->get_param( 'is_upe_enabled' );
-		$this->update_enabled_payment_methods( $payment_method_ids_to_enable, $is_upe_enabled );
+		$payment_method_ids_to_enable  = $this->get_payment_method_ids_to_enable( $request );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $payment_method_ids_to_enable. Keep variable name length under 20. (undefined)

(LongVariable)